data/json/items/armor/storage.json

```
@@ -509,8 +509,9 @@
     "color": "brown",
     "covers": [ "torso" ],
     "coverage": 8,
-    "encumbrance": 2,
-    "storage": "2 L",
+    "encumbrance": 3,
```

```suggestion
    "encumbrance": 2,
```
how about we keep it 2 as it's only when empty and the max encumbrance is much higher (6)?

---

That works, sure.